manifest/manifest.go

```diff
@@ -124,3 +125,9 @@ func ParseDir(chartDir string) ([]*Manifest, error) {
 
 	return files, filepath.Walk(dir, walker)
 }
+
+// IsKeeper returns true if a "helm-keep: true" annotation is present in manifest bytes.
+func IsKeeper(data []byte) bool {
+	var keep = regexp.MustCompile(`\"annotations\": \{(\n.*){0,10}\"helm-keep\": true`)
```

Comment:
I agree it's not as straightforward to implement if it's in a header instead.  Ease of implementation set aside, I'm torn on which is the "right" way to do this.  I don't think I have enough of an opinion on it yet.